README.md

```diff
@@ -1,11 +1,11 @@
 # axe-core
 
-[![Join the chat at https://gitter.im/dequelabs/axe-core](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/dequelabs/axe-core?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-[![Version](https://img.shields.io/npm/v/axe-core.svg)](https://www.npmjs.com/package/axe-core)
-[![Bower](https://img.shields.io/bower/v/axe-core.svg)](http://bower.io/search)
 [![License](https://img.shields.io/npm/l/axe-core.svg)](LICENSE)
-[![CircleCI Build](https://circleci.com/gh/dequelabs/axe-core/tree/develop.svg?style=svg)](https://circleci.com/gh/dequelabs/axe-core/tree/develop)
```

Comment:
No circle build anymore?

Reply:
Maybe because no code was change? Just guessing here.

Reply:
@marcysutton see my comment above - if you think we should keep the build status, what is the rationale?